src/sage/features/databases.py

```diff
@@ -262,7 +262,8 @@ def __init__(self, name='polytopes_db'):
 
 
 def all_features():
-    return [PythonModule('conway_polynomials'),
+    return [PythonModule('conway_polynomials', spkg='conway_polynomials', type='standard'),
+            PythonModule('matroid_database', spkg='matroid_database'),
```

Comment:
I'd suggest to make this a separate feature class `DatabaseMatroids`.

Then, just before you `import matroid_database`, use `DatabaseMatroids().require()`. This will give instructions to users how to install it.